docs: add keptn certificate manager infos (#652)
keptn/lifecycle-toolkit: README.md

process and export telemetry data. To install it, follow their [installation instructions](https://opentelemetry.io/docs/collector/getting-started/).
We also provide some more information about this in our [observability example](./examples/observability/).

The Lifecycle Toolkit includes a Mutating Webhook which requires TLS certificates to be mounted as a volume in its pod. The certificate creation

> is handled automatically by [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/klt-cert-manager/README.md), versions up to 0.5.0 have a hard dependency on cert-manager.

Reviewer: ```suggestion
is handled automatically by [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/klt-cert-manager/README.md). Versions up to 0.5.0 have a hard dependency on [cert-manager](https://cert-manager.io).
```

> is handled automatically by [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/klt-cert-manager/README.md). Versions up to 0.5.0 have a hard dependency on [cert-manager](https://cert-manager.io).

Reviewer 2: ```suggestion
is handled automatically by [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/klt-cert-manager/README.md). Versions 0.5.0 and earlier have a hard dependency on the [cert-manager](https://cert-manager.io).
```
